Xml.psm1

```diff
@@ -634,44 +635,48 @@ function ConvertFrom-Html {
     return $xml
 }
 
-
 function Convert-Xml {
     #.Synopsis
     #   The Convert-XML function lets you use Xslt to transform XML strings and documents.
     #.Description
     #   Documentation TODO
-    [CmdletBinding(DefaultParameterSetName="Xml")]
+    [CmdletBinding(DefaultParameterSetName="Xsl")]
```

Comment:
It's not the name I objected to, it looked like the parameter set that had been default was now secondary. After reading it a little more closely, that's not the case, I'll merge this.